docs/guide/migration.md

```diff
@@ -267,6 +267,7 @@ Vitest 4.0 removes some deprecated APIs, including:
 - `poolMatchGlobs` config option. Use [`projects`](/guide/projects) instead.
 - `environmentMatchGlobs` config option. Use [`projects`](/guide/projects) instead.
 - Reporter APIs `onCollected`, `onSpecsCollected`, `onPathsCollected`, `onTaskUpdate` and `onFinished`. See [`Reporters API`](/advanced/api/reporters) for new alternatives. These APIs were introduced in Vitest `v3.0.0`.
+- The `verbose` reporter now always prints tests cases even if the terminal is TTY (previously, it would fallback to the `default` reporter in that case).
```

Comment:
Does this mean that in TTY, `--reporter verbose` is equal to `--reporter default`? I don't think it is.

Reply:
It basically is. Most of the rendering is done by the default reporter in that case

Reply:
> Yeah, but it's done by `verbose: true`, not by the reporter

Which is basically the same.

In any way, we need to have the same view for TTY and non-TTY - either remove tree view (which is done here) or remove the long names view